src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -15,12 +15,20 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ProjectGuid>{9E50E7BF-CD6E-4269-A6DD-59FD0BD6C0FD}</ProjectGuid>
     <CommonPath Condition="'$(CommonPath)' == ''">..\..\Common\src</CommonPath>
-    <IsNETNativeLibrary Condition="'$(TargetGroup)' == 'netcore50'">true</IsNETNativeLibrary>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(IsNETNativeLibrary)' == 'true'">
+
+  <!-- Default configurations to help VS understand the options -->
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'dnxcore50_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'dnxcore50_Release|AnyCPU'" />
+
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
     <DefineConstants>$(DefineConstants);FEATURE_NETNATIVE</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(IsNETNativeLibrary)' != 'true'">
+  <PropertyGroup Condition="'$(TargetGroup)' != 'netcore50'">
```

Comment:
IsNetNativeLibrary is being set explicitly in our ToF wrapper projects' settings file when it detects it is being run against N -- so that it would trigger these conditional properties and constants.  So we may need to alter the TFS Tof WCF.settings.targets file to set $(TargetGroup) instead.  We could make the settings file set both properties first so that it works with both the unmodified OSS code as well as the modified OSS code.